src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
```suggestion
  constructor(name) {
    super(name);
    this.hidden = false;
  }
  
  hide() {
    this.hidden = true;
  }
```

Reply:
Also, how can this animal stop hiding? 🙂 Better to change `hidden` value to opposite here

Reply:
I was wrong, ignore this comment 🙃